Jenkinsfile

```diff
@@ -30,53 +30,21 @@ pipeline {
     disableConcurrentBuilds()
   }
   stages {
-    stage('build and stash bibdk code') {
-      agent {
-        docker {
-          label "devel9-head"
-          image "docker-dscrum.dbc.dk/d7-php7-builder:latest"
-          alwaysPull true
-        }
-      }
-      steps {
-        // Drush Make
-        sh """
-          drush make -v --working-copy --strict=0 --dbc-modules=$BRANCH_NAME --no-gitinfofile --contrib-destination=profiles/bibdk $DISTROPATH www
-        """
-        // Building CSS
-        dir('www/profiles/bibdk/themes/bibdk_theme/.npm') {
-          sh """
-            npm install
-            ./node_modules/gulp/bin/gulp.js build --production
-          """
-        }
-        // Stuffing a tar with the code.
-        sh """
-        tar -czf www.tar www
-        """
-        stash name: "www", includes: "www.tar"
-      }
-    }
 
     stage('Docker: Drupal Site') {
       agent {
         node { label 'devel9-head' }
       }
       steps {
         dir('docker/www') {
-          unstash "www"
-          sh """
-          tar -xf www.tar
-          """
           script {
-            docker.build("${DOCKER_REPO}/${PRODUCT}-www-${BRANCH}:${currentBuild.number}")
+            docker.build("${DOCKER_REPO}/${PRODUCT}-www-${BRANCH}:${currentBuild.number}", "--build-arg BRANCH=${BRANCH_NAME} .")
```

Comment:
Har du kørt denne kode? I Filmbilleder, Faktalink & Forfatterweb skal jeg køre koden sådan:
`docker.build("${DOCKER_REPO}/${PRODUCT}-www-${BRANCH}:${currentBuild.number}",
            "-f ./docker/www/Dockerfile --no-cache --build-arg BRANCH=${BRANCH_NAME} .")`
Læg mærke til -f variablen.

Reply:
Ah, jeg har ikke den dir (docker/www) med i FLFWFB. For ellers kunne jeg ikke få composer med i docker billedet.